package.json

```diff
@@ -30,7 +30,7 @@
     "electron-localshortcut": "^0.6.0"
   },
   "devDependencies": {
-    "electron-prebuilt": "0.37.2",
+    "electron-prebuilt": "^0.37.7",
```

Comment:
I like it locked down so I don't have to redownload a new Electron version each time I'm doing tests (They release very often).